eng/Directory.Build.Data.props

```diff
@@ -30,6 +30,8 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <!-- Avoid implicitly added preview feeds -->
+    <AddDotnetfeedProjectSource>false</AddDotnetfeedProjectSource>
```

Comment:
Looks like this won't be necessary after we move to preview 6, https://github.com/dotnet/core-sdk/pull/1561. However I'm good with this for now. Unless there is a strong reason I don't plan to move the toolset version until .NET Core 3 ships stable.